internal/evidence/pool.go

```diff
@@ -316,7 +333,16 @@ func (evpool *Pool) addPendingEvidence(ev types.Evidence) error {
 	}
 
 	atomic.AddUint32(&evpool.evidenceSize, 1)
-	return nil
+	evpool.Metrics.NumEvidence.Set(float64(evpool.evidenceSize))
+
+	// This should normally never be true
+	if evpool.eventBus == nil {
+		return fmt.Errorf("event bus is not configured")
+	}
```

Comment:
I don't think we should error. The event bus is, to me at least, an optional component of the evidence pool. `addPendingEvidence` should only fail if we failed to add evidence. If we don't want this to error silently then I'd suggest just logging it as `Debug`

Reply:
We discussed this here: https://github.com/tendermint/tendermint/pull/7802#discussion_r809692852 and perhaps indeed logging is a middle-ground solution?

Reply:
I missed that conversation. Yeah I think logging seems reasonable

Reply:
Done!